### PR TITLE
Create Yodlee API logic in Source Manager

### LINKED ISSUE #271: Source Manager: Add Yodlee api
Add yodlee api class which will serve for interacting with Yodlee.

### PR DESCRIPTION
**Pull request by bnenov**
1. Moved config package outside nordigen
2. Added Yodlee api logic and model classes
3. Added new properties in yaml file and ApplicationProperties config class

closes #271

### CODE REVIEW

File: source-manager/src/main/java/com/jointrivial/sourcemanager/yodlee/api/YodleeApi.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+            return accessTokenSerialized.getToken().getAccessToken();
+        }
+
+        return null;
+
+    }
+
+    /**
+     * Registers the user in Yodlee. Every use has to be registered with a login name
```
Comment (Semerkosa) [inline source-manager/src/main/java/com/jointrivial/sourcemanager/yodlee/api/YodleeApi.java:74]: Missing letter in 'Every use**R**' :)

**Review by Semerkosa — APPROVED** [review]
LGTM!
However, I would only suggest thinking of extracting the private methods from YodleeApi (access token, baseUrl or static URIs) to another base class/es.

**Review by ANeshkova — APPROVED** [review]

**Review by plm012 — APPROVED** [review]

**Review by GGerginov — APPROVED** [review]

File: source-manager/src/main/java/com/jointrivial/sourcemanager/yodlee/api/YodleeApi.java (hunk 1 above)
Comment (bnenov) [reply source-manager/src/main/java/com/jointrivial/sourcemanager/yodlee/api/YodleeApi.java]: Thanks for the note. the typo is fixed.

**Comment by bnenov** [comment]
> LGTM! However, I would only suggest thinking of extracting the private methods from YodleeApi (access token, baseUrl or static URIs) to another base class/es.

I agree to that but the only reason I left it like this is because this code is supposed to be a java library in order to be more reusable, but since creating a library will take more time, and this code will also be used in Asset service, I decided to have less files to copy to asset later.

**Pull request merged**